diambraArena/diambraEnvLib/diambra_pb2_grpc.py

```diff
@@ -16,45 +16,46 @@ def __init__(self, channel):
             channel: A grpc.Channel.
         """
         self.EnvInit = channel.unary_unary(
-                '/diambra.EnvServer/EnvInit',
-                request_serializer=diambra__pb2.EnvSettings.SerializeToString,
-                response_deserializer=diambra__pb2.EnvInfoAndIntData.FromString,
-                )
+            '/diambra.EnvServer/EnvInit',
+            request_serializer=diambra__pb2.EnvSettings.SerializeToString,
+            response_deserializer=diambra__pb2.EnvInfoAndIntData.FromString,
+        )
         self.Reset = channel.unary_unary(
-                '/diambra.EnvServer/Reset',
-                request_serializer=diambra__pb2.Empty.SerializeToString,
-                response_deserializer=diambra__pb2.Obs.FromString,
-                )
+            '/diambra.EnvServer/Reset',
+            request_serializer=diambra__pb2.Empty.SerializeToString,
+            response_deserializer=diambra__pb2.Obs.FromString,
+        )
         self.Step1P = channel.unary_unary(
-                '/diambra.EnvServer/Step1P',
-                request_serializer=diambra__pb2.Command.SerializeToString,
-                response_deserializer=diambra__pb2.Obs.FromString,
-                )
+            '/diambra.EnvServer/Step1P',
+            request_serializer=diambra__pb2.Command.SerializeToString,
+            response_deserializer=diambra__pb2.Obs.FromString,
+        )
         self.Step2P = channel.unary_unary(
-                '/diambra.EnvServer/Step2P',
-                request_serializer=diambra__pb2.Command.SerializeToString,
-                response_deserializer=diambra__pb2.Obs.FromString,
-                )
+            '/diambra.EnvServer/Step2P',
+            request_serializer=diambra__pb2.Command.SerializeToString,
+            response_deserializer=diambra__pb2.Obs.FromString,
+        )
         self.Close = channel.unary_unary(
-                '/diambra.EnvServer/Close',
-                request_serializer=diambra__pb2.Empty.SerializeToString,
-                response_deserializer=diambra__pb2.Empty.FromString,
-                )
+            '/diambra.EnvServer/Close',
+            request_serializer=diambra__pb2.Empty.SerializeToString,
+            response_deserializer=diambra__pb2.Empty.FromString,
+        )
 
 
 class EnvServerServicer(object):
     """The DIAMBRA service definition.
     """
 
     def EnvInit(self, request, context):
```

Comment:
@discordianfish these names are created automatically and inherited from the proto file. Can we keep them as they are?